kinematics/kinematics.c

```diff
@@ -30,23 +30,22 @@ kinematics_t kinematics_create(enum base k_base, float length_x, float length_y,
     new_kinematics->length_y     = length_y;
     new_kinematics->wheel_cir = wheel_radius * 2.0f * PI;;
 
-    if(k_base == TWO_WD)
+    switch(k_base)
     {
+    case TWO_WD:
         new_kinematics->total_wheels = 2;
-    }
-    if(k_base == FOUR_WD)
-    {
-        new_kinematics->total_wheels = 4;
-    }
-    if(k_base == ACKERMANN)
-    {
+	    break;

```

Comment:
!!! "tab"

Reply:
ok, this is not 'tab', pinch in when use 'swith(){case x: break}'